Add makefile targets for managing local dev FT log

AlCutter opened the pull request:
This PR:
  - Adds support for managing a local serverless log instance for local testing.
  - Creates a manifest file whenever the applet is built

To use the log, append a `log_applet` target to the make invocation, e.g.:

```
make trusted_applet log_applet
```

Note that `LOG_PUBLIC_KEY` and `LOG_PRIVATE_KEY` env variables are expected to be set to paths to key files, in keeping with the other key-related env vars.

jiggoha commented:
I think this will break the Dockerfile build since we now need Go. I think we need to change the base image in the Dockerfile.

AlCutter commented:
> I think this will break the Dockerfile build since we now need Go. I think we need to change the base image in the Dockerfile.

Ah, the `go run` in the `$(APP).elf` target...

Ok, I've tweaked the Dockerfile and the Makefile a bit:
  - Docker image -> Alpine+go (plus some `apk` fixups)
  - Makefile -> I've added a new "no sign" target, and extracted the creation of the manifest into its own target and added a dep so it gets called when building the applet.

jiggoha commented:
Great, thanks!
